src/web_socket_connection.rs

```diff
@@ -70,9 +75,34 @@ impl StreamHandler<Result<ws::Message, ws::ProtocolError>> for MyWs {
                         let op: &str = op.extract().unwrap();
 
                         return ctx.text(op);
+                        // return ctx.text(op);
                     }),
-                    PyFunction::CoRoutine(_handler) => {
-                        println!("Async functions are not supported in WS right now.");
+                    PyFunction::CoRoutine(handler) => {
+                        println!("{:?}", handler);
+                        let fut = Python::with_gil(|py| {
+                            let handler = handler.as_ref(py);
+                            let coro = handler.call0().unwrap();
+                            pyo3_asyncio::tokio::into_future(coro)
+                        });
```

Comment:
Hi @awestlake87 ,

I hope everything is going well.

I trying to convert a python coroutine to a rust future and I am getting the following error:


```
sys:1: RuntimeWarning: coroutine 'connect' was never awaited
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
thread 'actix-rt|system:0|arbiter:0' panicked at 'called `Result::unwrap()` on an `Err` value: PyErr { type: <class 'RuntimeError'>, value: RuntimeError('no running event loop'), traceback: None }', src/web_socket_connection.rs:101:33
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I am using the same method for conversion as I am doing here: https://github.com/sansyrox/robyn/blob/main/src/processor.rs#L84

I am unable to figure out why this is giving this error.

Can you please help? 

Thanks in advance.